client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx

```diff
@@ -520,17 +520,21 @@ function StatsCommercialFlowOptOutForm( {
 					) }
 					{ isClassificationInProgress && ! errorMessage && (
 						<p className={ `${ COMPONENT_CLASS_NAME }__error-msg` }>
-							{ translate( 'We are verifying your site. Please come back later…' ) }
+							{ translate(
+								'We are working on verifying your site… Please come back in about 30 minutes. You will have an option to contact support when the process is finished.'
+							) }
 						</p>
 					) }
 					{ ! isClassificationInProgress &&
 						commercialClassificationLastRunAt > 0 &&
 						! errorMessage && (
-							<p className={ `${ COMPONENT_CLASS_NAME }__error-msg` }>
-								{ translate(
-									'We have finished verify your site. If you still think this is an error, please contact our support.'
-								) }
-							</p>
+							<>
+								<p className={ `${ COMPONENT_CLASS_NAME }__error-msg` }>
+									{ translate(
+										'We have finished verifying your site. If you still think this is an error, please contact support by clicking the button above. '
```

Comment:
```suggestion
										'We have finished verifying your site. If you still think this is an error, please contact support by clicking the button above.'
```

A redundant space after the period.